### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by numel007**
I put stuff in the thing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+      "date": "2021-02-18"
+    }
+  ]
+}
+```
+
+## License
+`StonkScraper` is available under the MIT license. See the [LICENSE](https://github.com/BoringStocks/StonkScraper/blob/dev/LICENSE) file for more info.
+
+# Disclaimer
```
Comment (pondorasti) [inline README.md]: Is there a reason why this header is bigger in size than the other ones like `License`?

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
-# StonkScraper
\ No newline at end of file
+# __StonkScraper__
```
Comment (pondorasti) [inline README.md:1]: I think we should also add a note that the project is still under development.

File: README.md (hunk 1 above)
Comment (numel007) [reply README.md]: Nope, my knowledge of markdown broke down (no pun intended) and I thought a single hash was smaller than 2 hashes.

**Review by pondorasti — APPROVED** [review]
Looks great!

**Pull request merged**